jest.config.ts

```diff
@@ -1,10 +1,11 @@
-module.exports = {
+import { JestConfigWithTsJest } from "ts-jest";
+
+export default {
   roots: ["<rootDir>"],
   clearMocks: true,
   testMatch: ["**/(__tests__|src)/**/?(*.)+(spec|test).+(ts|tsx|js)"],
-  transform: {
-    "^.+\\.(ts|tsx)?$": "ts-jest",
-  },
+  preset: "ts-jest/presets/js-with-ts",
+  transformIgnorePatterns: [],
```

Comment:
let's add a comment as to why.

Reply:
53b454b1bf7188d14e2e793db709c96a74a37cd4